docker/build/diff-sitemaps.sh

```diff
@@ -1,13 +1,45 @@
+#!/bin/bash
+
+
 CURRENT_DATE=$(date +%Y%m%d)
+ENVIRONMENT=production
```

Comment:
```suggestion
if [ -z "$ENVIRONMENT"];
then
  ENVIRONMENT=production
fi 
```

how about only setting a default if it's not set. That way it behaves the same as the `Makefile`.

Reply:
actually, maybe we could rename it to `OBJECTIV_ENVIRONMENT`